src/doc/unstable-book/src/language-features/doc-spotlight.md

```diff
@@ -5,8 +5,8 @@ The tracking issue for this feature is: [#45040]
 The `doc_spotlight` feature allows the use of the `spotlight` parameter to the `#[doc]` attribute,
 to "spotlight" a specific trait on the return values of functions. Adding a `#[doc(spotlight)]`
 attribute to a trait definition will make rustdoc print extra information for functions which return
-a type that implements that trait. This attribute is applied to the `Iterator`, `io::Read`, and
-`io::Write` traits in the standard library.
+a type that implements that trait. This attribute is applied to the `Iterator`, `io::Read`,
+`io::Write`, and `Future` traits in the standard library.
```

Comment:
Well, I'm not sure if it's useful to mention all the traits. Maybe add "for example" or something along the line so we don't have to update it any time we add spotlight on another trait?

Reply:
Since there are very few spotlight in standard library, I don't see why we shouldn't add all of them. We can count them within one hand.

Reply:
For now, who knows what will happen in the future?

Reply:
In case that happens we can ignore it if it increase a lot. Here it also let increase the readers knowledge so they could search for special examples, at least it is not added for nothing.

Reply:
Or we can handle it now so it's done and we can sleep with peace in mind.

Reply:
@GuillaumeGomez Your call. Merge or no merge?

Reply:
With what I suggested, yes. As is, no.

Reply:
Done. I did what you suggested.

Reply:
Thanks!